src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -115,6 +115,7 @@ public class SnowflakeSinkConnectorConfig {
 
   // This is the streaming bdec file version which can be defined in config
   // NOTE: Please do not override this value unless recommended from snowflake
+  // TODO @rcheng: remove?
   public static final String SNOWPIPE_STREAMING_FILE_VERSION = "snowflake.streaming.file.version";
```

Comment:
Since we only allow bdec version 3 in the next release, can we remove this config? Or just ignore the override for backcompat concerns

Reply:
Lets keep it, might be useful in future, we anyways havent created public documentation for this.